.bazelci/presubmit.yml

```diff
@@ -45,3 +45,14 @@ platforms:
     - "--extra_execution_platforms=@bazel_toolchains//configs/ubuntu16_04_clang/latest:platform_docker"
     - "--host_platform=@bazel_toolchains//configs/ubuntu16_04_clang/latest:platform_docker"
     - "--platforms=@bazel_toolchains//configs/ubuntu16_04_clang/latest:platform_docker"
+  rbe_exec_properties:
```

Comment:
add short doc comment above explaining what is being tested here differently from above.

Reply:
Done.